opsdroid/connector/websocket/__init__.py

```diff
@@ -18,13 +18,12 @@
 class ConnectorWebsocket(Connector):
     """A connector which allows websocket connections."""
 
-    def __init__(self, config):
+    def __init__(self, config, opsdroid=None):
         """Create the connector."""
-        super().__init__(config)
+        super().__init__(config, opsdroid=opsdroid)
         _LOGGER.debug("Starting Websocket connector")
         self.name = "websocket"
         self.config = config
-        self.opsdroid = None
```

Comment:
Do you know this is the reason why we are getting the `NoneType` issue within docker?

Reply:
I bet that wherever `ConnectorWebsocket` is being initialized it's not being given the opsdroid argument that it needs now, so it's defaulting to None.

Reply:
I think I found it in core